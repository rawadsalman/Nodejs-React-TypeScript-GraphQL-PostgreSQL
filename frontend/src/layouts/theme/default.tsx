// rgb(17, 24, 57) blau
// rgb(247, 230, 153)	gelb

const defaultTheme = {
  colors: {
    primary: 'rgb(255, 0, 130)',
    secondary: '#1e272e',
    green: '#16a085',
    darkBlue: '#001529',
    almostWhite: '#f0f0f0',
    almostGrey: '#eaeaea',
    yellow: '#f6e58d',
    paleRed: '#ff7675',
  },
  modal: {
    width: 720,
  },
}

export default defaultTheme
