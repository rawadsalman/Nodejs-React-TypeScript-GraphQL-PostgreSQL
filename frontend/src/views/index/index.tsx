import React from 'react'
import { StyledContent } from '../styles'

const IndexView: React.FC = () => {
  return (
    <StyledContent>
      <p>Hey there</p>
    </StyledContent>
  )
}

export default IndexView
