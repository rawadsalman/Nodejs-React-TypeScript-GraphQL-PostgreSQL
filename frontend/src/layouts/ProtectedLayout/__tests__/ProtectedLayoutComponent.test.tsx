import { render } from '@testing-library/react'
import { ThemeProvider } from '@emotion/react'
import React from 'react'
import { MemoryRouter as Router } from 'react-router-dom'
import { defaultTheme } from '../../../layouts/theme/index'
import DefaultLayout from '../index'

describe('Protected Layout', () => {
  it('matches Snapshot', () => {
    const layout = render(
      <ThemeProvider theme={defaultTheme}>
        <Router initialEntries={['/']}>
          <DefaultLayout />
        </Router>
      </ThemeProvider>
    )
    expect(layout).toMatchSnapshot()
  })
})
