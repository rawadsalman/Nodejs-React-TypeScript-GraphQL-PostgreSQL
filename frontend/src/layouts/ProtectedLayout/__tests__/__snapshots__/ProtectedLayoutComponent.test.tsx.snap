// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Protected Layout matches Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-w3ihf7-SidebarLayoutWrapper evhn4ae2"
      >
        <aside
          class="ant-layout-sider ant-layout-sider-dark css-fg5ay8-SidebarWrapper evhn4ae0"
          style="flex: 0 0 78px; max-width: 78px; min-width: 78px; width: 78px;"
        >
          <div
            class="ant-layout-sider-children"
          >
            <div
              class="css-17qfunh-FlexBoxDiv e1x7gwan2"
            >
              <div>
                <div
                  class="css-va9r8i-TitleDiv e1x7gwan1"
                >
                  <a
                    href="/"
                  >
                    <h2
                      class="css-1br8yq8-LogoName e1x7gwan0"
                    >
                      Space
                      <br />
                      Goats
                    </h2>
                  </a>
                  <hr />
                </div>
                <div
                  class=" css-1imfco6-NavItemOuterDiv ek60gb91"
                >
                  <div
                    class=" css-m82ati-NavItemInnerDiv ek60gb92"
                  >
                    <svg>
                      iconmonstr-building-46.svg
                    </svg>
                  </div>
                  <div
                    class="flyout css-1ev6f10-NavItemFlyout ek60gb90"
                  >
                    <span>
                      companies
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </aside>
        <section
          class="ant-layout css-crbcnx-PageWrapper evhn4ae1"
        >
          <main
            class="ant-layout-content css-hzyx1i-StyledContent e3kcq9h1"
          >
            <p>
              Hey there
            </p>
          </main>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-w3ihf7-SidebarLayoutWrapper evhn4ae2"
    >
      <aside
        class="ant-layout-sider ant-layout-sider-dark css-fg5ay8-SidebarWrapper evhn4ae0"
        style="flex: 0 0 78px; max-width: 78px; min-width: 78px; width: 78px;"
      >
        <div
          class="ant-layout-sider-children"
        >
          <div
            class="css-17qfunh-FlexBoxDiv e1x7gwan2"
          >
            <div>
              <div
                class="css-va9r8i-TitleDiv e1x7gwan1"
              >
                <a
                  href="/"
                >
                  <h2
                    class="css-1br8yq8-LogoName e1x7gwan0"
                  >
                    Space
                    <br />
                    Goats
                  </h2>
                </a>
                <hr />
              </div>
              <div
                class=" css-1imfco6-NavItemOuterDiv ek60gb91"
              >
                <div
                  class=" css-m82ati-NavItemInnerDiv ek60gb92"
                >
                  <svg>
                    iconmonstr-building-46.svg
                  </svg>
                </div>
                <div
                  class="flyout css-1ev6f10-NavItemFlyout ek60gb90"
                >
                  <span>
                    companies
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </aside>
      <section
        class="ant-layout css-crbcnx-PageWrapper evhn4ae1"
      >
        <main
          class="ant-layout-content css-hzyx1i-StyledContent e3kcq9h1"
        >
          <p>
            Hey there
          </p>
        </main>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
